st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -141,7 +223,9 @@ class ActionExecutionsController(ActionExecutionsControllerMixin, ResourceContro
 
     children = ActionExecutionChildrenController()
     attribute = ActionExecutionAttributeController()
+    re_run = ActionExecutionReRunController()
```

Comment:
I think we should have one or another and not both ("There should be one-- and preferably only one --obvious way to do it." :P).